backend/python/app/tests/helpers/story_translation_helpers.py

```diff
@@ -62,6 +62,33 @@ def assert_story_translation_equals_model(
     assert story_translation_response["num_content_lines"] == len(translation_contents)
 
 
+def assert_story_translation_test_equals_model(
```

Comment:
AWWWWWWWWWWWWWWWWWWWWWWWWWWWWW YEAHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH WE LOVE SUM GUD UNIT TEST HELPERS

Reply:
oh it's just bc of the extra fields like `translation_contents`, `num_translated_lines`,and `num_approved_lines` but I can try using `assert_story_translation_equals_model` also